app/controllers/api/v1/users_controller.rb

```diff
@@ -2,4 +2,32 @@ class Api::V1::UsersController < ApplicationController
   def index
     render json: UserSerializer.new(User.all)
   end
+
+  def show
+    render json: UserSerializer.new(User.find(params[:id]))
+  end
+
+  def create
+    user = User.new(user_params)
+    if user.save
+      render json: UserSerializer.new(User.create(user_params)), status: 201
+    else
+      user_message = FailedUser.new(user)
```

Comment:
Will trigger the failed user poro if user information was not filled in correctly